runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerConfig.java

```diff
@@ -90,4 +90,4 @@ default boolean syncPanel()
 	{
 		return true;
 	}
-}
\ No newline at end of file
+}
```

Comment:
Remove this eof line addition now that there are no other config changes

Reply:
Done.  I squashed my 2 commits together - assuming that the visual button is a better approach.